src/app/interfaces/menusInterfaces.ts

```diff
@@ -132,6 +132,8 @@ export interface MenuLink extends MenuItem {
   kind: "MenuLink";
   /**
    * The URL or fragment this link points to
+   * TODO Add input for different root urls so that Href can choose where to
+   * route
```

Comment:
Remove: I don't think that's the right design - there should never be any ambiguity in a URI, as in this should be normalized by controllers/models before this point